src/BloomExe/Edit/EditingModel.cs

```diff
@@ -546,6 +549,10 @@ void OnPageSelectionChanged(object sender, EventArgs e)
 					_view.UpdateThumbnailAsync(_previouslySelectedPage);
 				}
 				_previouslySelectedPage = _pageSelection.CurrentSelection;
+
+				// BL-2339: remember last edited page
+				_pageSelection.CurrentSelection.Book.UserPrefs.MostRecentPage = _pageSelection.CurrentSelection.GetIndex();
```

Comment:
Is there any scenario (e.g., there are many ways like switching tabs and exiting Bloom to leave the current book; deleting and re-ordering pages; insert page) where this might get called and _pageSelection.CurrentSelection is null or not a valid, current page?